float32 angle
float32 interval_angle
float32 snap_interval
float32 zvel
---
uint8 status
sensor_msgs/Image image

uint8 STARTED=0
uint8 IN_PROGRESS=1
uint8 ERROR=2
uint8 STOPPED=3
